airflow/task_runner/base_task_runner.py

```diff
@@ -119,7 +120,7 @@ def run_command(self, run_with, join_args=False):
         # Start daemon thread to read subprocess logging output
         log_reader = threading.Thread(
             target=self._read_task_logs,
-            args=(proc.stdout,),
```

Comment:
@aoen You mean `decode('utf-8')` as the `readline()` will provide a `bytes` object. 
```
In [1]: f = open('test', 'rb')

In [2]: line = f.readline()

In [3]: line
Out[3]: b'abcdef\n'

In [4]: line.decode('utf-8')
Out[4]: 'abcdef\n'
```
But as you suggest, adding a `decode('utf-8')` solves the issue on my side.